monitoring/mongodb/alerts.yaml

```diff
@@ -97,20 +97,65 @@ groups:
   - alert: ReplicationLagWarning
     expr: |
       label_replace(
-        max(
-            max(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"})
-                by(pod, rs_nm)
-            - ignoring(member_idx) group_right
-            min(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="SECONDARY"})
-                by(pod, rs_nm, member_idx)
-        ) by(member_idx, rs_nm) / 1000 > 30
-      , "member_idx", "$1", "member_idx", "(${service}[^.]*)\\.${service}.*")
-    for: 10m
+        max by(rs_nm, member_idx) (
+          (
+            max by(pod, rs_nm) (
+              mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"}
+            ) - ignoring(member_idx) group_right
```

Comment:
are there multiple `member_idx` by `pod` ?
if not, a more 'readable' way would be to just do `max by(pod, member_idx, rs_nm) - min by (pod, member_idx, rs_nm)` ?

Reply:
To me with this suggestion, it tries to match the PRIMARY’s data to each SECONDARY’s `member_idx`, but the PRIMARY only has one index: it won't match with the secondaries and give empty data.
The original approach (that I am reusing in this PR) ignores `member_idx` to compare the PRIMARY to all SECONDARY members at once.

(in other words, the suggestion doesn't work)

Reply:
right, that was the first part of the comment: i was not sure how the metrics where presented (i.e. each pod exposes his view of the state of each member)